bgpd/bgp_mplsvpn.c

```diff
@@ -889,10 +901,22 @@ void delete_vrf_tovpn_sid_per_af(struct bgp *bgp_vpn, struct bgp *bgp_vrf,
 	if (tovpn_sid_index != 0 || tovpn_sid_auto)
 		return;
 
+	if (bgp_vrf->vrf_id == VRF_UNKNOWN) {
+		if (debug)
+			zlog_debug("%s: vrf %s: vrf_id not set, can't set zebra vrf label",
+				   __func__, bgp_vrf->name_pretty);
+		return;
+	}
+
```

Comment:
when we are going to hit this?

Reply:
In some cases, `vrf_id` can be `VRF_UNKNOWN`. This can happen, for example, when you configure a BGP instance for a VRF, but the VRF does not yet exist in the data plane.

Reply:
thanks